.eslint-dictionary.json

```diff
@@ -132,8 +156,11 @@
   "epath",
   "errno",
   "esri",
+  "ession",
+  "etag",
   "execa",
   "executables",
+  "exprs",
```

Comment:
yes, this was for an expression tokenizer. I have removed exprs and added tokenizer since the original variable name was not super clear anyways. https://github.com/aws-amplify/amplify-cli/blob/2831c9e1ad3c199fa3406933c978982bc0c6aa2a/packages/amplify-category-function/src/provider-utils/awscloudformation/utils/cronExpression.ts#L125